docs/configuration-files.md

```diff
@@ -224,6 +226,64 @@ rules:
     severity: ERROR
 ```
 
+### `metavariable-comparison`
+
+The `metavariable-comparison` operator compares metavariables against a basic [Python comparison](https://docs.python.org/3/reference/expressions.html#comparisons)
+expression. This is useful for filtering results based on a [metavariable's](/docs/pattern-features.md#metavariables) value.
+
+**Example**
+
+The `metavariable-comparison` operator is a mapping which requires the
+`metavariable` and `comparison` keys. It can be combined with other pattern operators:
+
+```yaml
+rules:
+  - id: superuser-port
+    patterns:
+      - pattern: set_port($ARG)
+      - metavariable-comparison:
+          metavariable: '$ARG'
```

Comment:
why the quote around the metavariable? we don't put quote to express patterns.

Reply:
Also why specify the metavariable ... just export all the metavariables in the current matched state and pass it to semgrep-core -eval.
Also this prevents to use multiple metavariables comparisons as $X < $Y.

Reply:
Specifying the metavariable allows for the `base` and `strip` behavior. We could do something like make `metavariables`, `bases`, and `strips` all lists or something so we can specify multiple, but I went with the simplest approach here.